.gitignore

```diff
@@ -13,3 +13,4 @@
 /Licenses.toml
 /licenses
 *.run
+/my/
```

Comment:
Do you also want this ignored by `.dockerignore`? That controls what is and isn't copied into package and image builds as part of the build context.

Reply:
I was unaware of this (TIL--thanks!). Yes, it should be excluded there as well.